generators/app/templates/src/app/components/RadioGroup/components/RadioOption.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function RadioOption({
+  className,
+  inputClassName,
+  labelClassName,
+  value,
+  label,
+  name,
+  checked,
+  disabled,
+  onChange,
```

Comment:
I'd use a `...rest` prop so `input` becomes a bit more customizable. This way, `value`, `name`, `checked`, `disabled` and `onChange` don't need to be explicitly declared 🤗